tensorflow_probability/python/sts/smooth_seasonal_test.py

```diff
@@ -31,6 +31,22 @@
 
 class _SmoothSeasonalStateSpaceModelTest(object):
 
+  def test_period_and_frequency_multipliers_tensor(self):
```

Comment:
nit: we often try to make test names a (semi-) readable phrase describing the condition being tested. e.g., here I might suggest `test_accepts_tensor_valued_period_and_frequency_multipliers`.

Reply:
fixed in 7ae7cec9cdb082eb9a9248f289e7aceedb31e5d6